blueocean-dashboard/src/main/js/components/PullRequests.jsx

```diff
@@ -70,27 +79,32 @@ export class PullRequests extends Component {
             }
 
             config.pipeline = pipelineName;
-            this.props.fetchBranchesIfNeeded(config);
         }
     }
 
     render() {
-        const { branches } = this.props;
-
         if (this.state.unsupportedJob) {
             return (<NotSupported />);
         }
 
+        const { pullRequests } = this.props;
+
         // early out
-        if (!branches) {
+        if (!pullRequests) {
```

Comment:
@michaelneale , this is because pull requests are now fetched separately from branches to be more clear, I suppose